bin/cron/ops_data_pull

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+
+require_relative '../../dashboard/config/environment'
+require 'cdo/properties'
+include ApplicationHelper
+include UsersHelper
+
+pd_scripts = Script.all.select { |script| !!script.pd }
+pd_scripts.each do |script|
+  # Get all users with any activity in the script
+  users = User.joins('left join user_scripts on user_scripts.user_id = users.id').where(user_scripts: {script_id: script})
```

Comment:
I believe you should be to avoid the explicit join by adding a has_many :through declaration to the Script class, then this could just be "users = script.users".  (Can discuss in more detail if that's not clear.)